build.sbt

```diff
@@ -118,7 +118,6 @@ val sharedSettings = Seq(
     setNextVersion,
     commitNextVersion,
     ReleaseStep(action = Command.process("sonatypeReleaseAll", _)),
-    pushChanges
   ),
```

Comment:
todo: I will make this depend conditional on if we are pushing a snapshot or not